packages/kit/src/runtime/server/page/load_node.js

```diff
@@ -325,8 +323,15 @@ export async function load_node({
 		loaded = await module.load.call(null, load_input);
 
 		if (!loaded) {
+			// TODO do we still want to enforce this now that there's no fallthrough?
 			throw new Error(`load function must return a value${options.dev ? ` (${node.entry})` : ''}`);
 		}
+
+		// TODO remove for 1.9
```

Comment:
this typo exists a few places below as well
```suggestion
		// TODO remove for 1.0
```